src/components/sidebar.js

```diff
@@ -169,13 +170,28 @@ class Sidebar extends React.Component {
       );
     }
 
+    const username = current_user.user.username;
+    const test = RegExp(`user/${username}\/?`, 'i');
```

Comment:
1. you escape second `/` but not the first one. why?
2. are you sure we need `'i'` here?